Ubuntu/master.sh

```diff
@@ -134,13 +150,14 @@ helm version
 
 echo "4. Initialize kubernetes cluster:"
 kubeadm init --pod-network-cidr=192.168.0.0/16
+rm -rf $HOME/.kube
```

Comment:
After this step it may be great if we can check to ensure even kubelet is using systemd, or just force it by injecting our own env file.